client/src/lib/components/DatasetForm/DatasetForm.svelte

```diff
@@ -343,6 +380,36 @@
         </p>
       {/if}
     </fieldset>
+  </div>
+  <h2 class="fr-mb-5w">Mot-clés thématiques</h2>
+  <div class="form--content fr-mb-8w">
+    <div
+      class={`fr-input-group fr-mt-8w ${
+        hasTagsError($errors.tags) ? "fr-input-group--error" : ""
+      } `}
+    >
+      <label class="fr-label" for="tags">
+        Mot-clés <RequiredMarker />
+        <span class="fr-hint-text" id="technicalSource-desc-hint">
```

Comment:
Attention à bien faire correspondre les `id` pour le hint et l'error

```suggestion
        <span class="fr-hint-text" id="tags-desc-hint">
```